specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -359,32 +344,32 @@ export function StatsPage(): JSX.Element | null {
         <span className="-ml-2 flex-1" />
         {pageLastUpdated !== undefined && (
           <span>
-            {`${statsText.lastUpdated()} `}
+            {`${statsText.lastRefreshed()} `}
             <DateElement date={pageLastUpdated} />
           </span>
         )}
-        <Button.Blue
+        <Button.Gray
           onClick={(): void => {
             cleanMaybeFulfilled();
-            setLayout((layout) =>
+            getSourceLayoutSetter(activePage.isShared)((layout) =>
```

Comment:
Why not just say `const setCurrentLayout = getSourceLayoutSetter(activePage.isShared);` and then use `setCurrentLayout` where needed?

That way, you can even get rid of `getSourceLayoutSetter` since it would only have to be called once